src/ui/MessageContent/index.tsx

```diff
@@ -365,6 +365,7 @@ export default function MessageContent({
               <EmojiReactions
                 userId={userId}
                 message={message as UserMessage | FileMessage}
+                channel={channel}
```

Comment:
Cant we use hook?

Reply:
MessageContent is used under the Channel and Thread modules, do you have any good ideas to support both?

Reply:
yeah, I can share in chat ~

Reply:
Let me apply it in the different PR